tests/Fixer/FunctionNotation/NullableTypeDeclarationForDefaultNullValueFixerTest.php

```diff
@@ -753,63 +771,22 @@ public static function provideFix82Cases(): iterable
     }
 
     /**
-     * @dataProvider provideFix82InverseCases
+     * @param iterable<array{string, 1?: string}> $cases
      *
-     * @requires PHP 8.2
+     * @return iterable<array{string, null|string, 2?: array<string, bool>}>
      */
-    public function testFix82Inverse(string $expected, ?string $input = null): void
-    {
-        $this->fixer->configure(['use_nullable_type_declaration' => false]);
-        $this->doTest($expected, $input);
-    }
-
-    public static function provideFix82InverseCases(): iterable
+    private static function createBothWaysCases(iterable $cases): iterable
     {
-        yield from TestCaseUtils::swapExpectedInputTestCases(self::provideFix82Cases());
+        foreach ($cases as $key => $case) {
+            yield $key => $case;
 
```

Comment:
let's have the safeguard

```suggestion

            if (count($case) > 1) {
                throw new \BadMethodCallException(sprintf('Method "%s" does not support handling "configuration" input yet, please implement it.', __METHOD__));
            }

```